h2/events.py

```diff
@@ -136,6 +136,51 @@ def __repr__(self):
         )
 
 
+class ResponseSent(object):
```

Comment:
These are just ResponseReceived and TrailerReceived redressed with different names. I omitted a few of the extra fields, because we don’t have the logic to fill them in. (Unless I missed something.) Was that right?